package.nls.json

```diff
@@ -1,6 +1,7 @@
 {
     "staticWebApps.createStaticWebApp": "Create Static Web App...",
     "staticWebApps.createStaticWebAppAdvanced": "Create Static Web App... (Advanced)",
+    "staticWebApps.createStaticWebAppFromTemplate": "Create Static Web App from template...",
```

Comment:
nit: What do you think about `Create Static Web App... (Template)` or `Create Static Web App from Template...` Somehow the lower case t really bugs me aesthetically lol

Reply:
Changed to `Create Static Web App from Template...`.